### PR TITLE
Add StretchedKT to fit functions

### PR DESCRIPTION
**Pull request by hankwustfc**
Description of work.

Add StretchedKT to Mantid fit functions.

To test:

1.Test if the documentation is added
2.Test if the fit function is working
3.Test the function with the unit test file

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by AnthonyLim23** [comment]
release notes will be #26827

File: docs/source/fitting/fitfunctions/StretchedKT.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+===========
+
+.. index:: StretchedKT
+
+Description
+-----------
+
+For observation of muon spin-relaxation (:math:`\mu`SR) spectra with lineshapes lying between a Guassian and a Lorentzian,
+
+.. math:: A(t)=A_0\left(\frac13+\frac23\left(1-(\sigma t)^\beta\right)e^{-\frac{(\sigma t)^\beta}{\beta}}\right)
```
Comment (AnthonyLim23) [inline docs/source/fitting/fitfunctions/StretchedKT.rst]: the `frac13` etc will not work

File: docs/source/fitting/fitfunctions/StretchedKT.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+where,
+
+:math:`\sigma` is KuboToyabe decay rate,
+
+:math:`A_0` is the amplitude,
+
+:math:`\beta` is the stretching exponential and it ranges from 1 (neglible nuclear field) and 2 (strong nuclear field). 
+
+.. figure:: /images/StretchedKT.png
```
Comment (NickDraper) [inline docs/source/fitting/fitfunctions/StretchedKT.rst]: This should be replaced by a plot call

**Review by NickDraper — CHANGES_REQUESTED** [review]

**Review by NickDraper — APPROVED** [review]

**Comment by NickDraper** [comment]
code test and docs ok

**Pull request merged**